packages/api/src/shared/notifications.ts

```diff
@@ -57,6 +57,10 @@ export const capture = {
     Sentry.setUser(user);
   },
 
+  setExtra: (user: UserData): void => {
+    Sentry.setExtra("userId", user.id);
+  },
```

Comment:
Let's make this more flexible so we can use it in other places that are setting extras that are not `UserData`, please.

Something like this:

```ts
  setExtra: (extra: Record<string, unknown>): void => {
    Object.entries(extra).forEach(([key, value]) => Sentry.setExtra(key, value));
  },
```

Note: I inferred `Record<string, unknown` by looking at the type definition/code of `Sentry.setExtra`.

Reply:
This makes sense! Gonna take a mental note of this for the future! 🙏